full-service/src/db/encryption_indicator.rs

```diff
@@ -0,0 +1,177 @@
+// Copyright (c) 2020 MobileCoin Inc.
+
+//! DB impl for the Locked model.
```

Comment:
I generally expect the word "locked" to be associated with a mutex, but here it seems to mean something like "encrypted-at-rest" or "the user has authenticated".